### DISCUSSION TITLE
Pets Prestígio

### DISCUSSION THREAD

**Discussion by ghost** [leogianfagna/IssueTracker#909] [Ideia]
### Servidor afetado

Henesys

### Qual o propósito da ideia?

Essa alteração permitiria que mais jogadores pudessem ter seu primeiro pet e talvez futuramente adquirir outros, retiraria o conceito pay to win estabelecido atualmente e define uma nova meta aos jogadores.

### Descrição da ideia

Com a chegada dos pets no Henesys, muitos jogadores que não têm condições de comprar um sugeriram que houvesse uma maneira de obter um gratuitamente.

Pensando nisso, gostaria de sugerir que o sistema de prestígio, em especial a habilidade de domesticação, seja ajustado para oferecer um pet exclusivo a cada nível de prestígio, ou um pet no início, com diferentes estilos nos níveis seguintes.

**Comment by leogianfagna** [comment]
Como obteria a calça?

**Reply by ghost** [reply]
Esta recompensa poderia ser repassada a uma das novas habilidades Mcmmo (Tridente/Clava/Besta)

**Reply by ghost** [reply]
Especialmente na habilidade de tridente, uma vez que fazer skin para tridente é extremamente difícil, pois quando ele é arremessado, se torna uma entidade.. uma calça seria mais apropriada como recompensa desse prestígio.

**Reply by leogianfagna** [reply]
faz sentido. acho que ainda vale a pena tentar uma skin para tridente e ver se não fica muito ruim. se ficar ruim e não for ter, é uma ótima opção
